### PR TITLE
actions

### PR DESCRIPTION
**Pull request by miiu96**
Added actions for `go build` and for running unit tests

### CODE REVIEW

File: .github/workflows/pr-build.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+          if [ -f Gopkg.toml ]; then
+              curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+              dep ensure
+          fi
+      - name: Build
+        run: go build
```
Comment (bogdan-rosianu) [inline .github/workflows/pr-build.yml]: add empty line at the end of the file

File: .github/workflows/pr-tests.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          go-version: ^1.15.6
+        id: go
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+      - name: Unit Test
```
Comment (bogdan-rosianu) [inline .github/workflows/pr-tests.yml:26]: Unit tests

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-build.yml (hunk 1 above)
Comment (miiu96) [reply .github/workflows/pr-build.yml]: done.

File: .github/workflows/pr-tests.yml (hunk 2 above)
Comment (miiu96) [reply .github/workflows/pr-tests.yml]: done.

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**